apps/app/src/test/scala/org/lfdecentralizedtrust/splice/integration/tests/TrustSpecificScanConnectionIntegrationTest.scala

```diff
@@ -0,0 +1,222 @@
+// Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+package org.lfdecentralizedtrust.splice.integration.tests
+
+import cats.data.NonEmptyList
+import com.digitalasset.canton.logging.SuppressionRule
+import com.digitalasset.canton.{HasActorSystem, HasExecutionContext}
+import org.lfdecentralizedtrust.splice.integration.EnvironmentDefinition
+import org.lfdecentralizedtrust.splice.integration.tests.SpliceTests.IntegrationTest
+import org.lfdecentralizedtrust.splice.util.{SvTestUtil, WalletTestUtil}
+import org.lfdecentralizedtrust.splice.config.ConfigTransforms
+import org.apache.pekko.http.scaladsl.model.Uri
+import org.lfdecentralizedtrust.splice.scan.admin.api.client.BftScanConnection.BftScanClientConfig
+import org.slf4j.event.Level
+import com.digitalasset.canton.config.NonNegativeFiniteDuration
+
+class TrustSpecificScanConnectionIntegrationTest
+    extends IntegrationTest
+    with WalletTestUtil
+    with SvTestUtil
+    with HasExecutionContext
+    with HasActorSystem {
+
+  override protected def runEventHistorySanityCheck: Boolean = false
+
+  override def environmentDefinition: SpliceEnvironmentDefinition =
+    EnvironmentDefinition
+      .fromResources(Seq("simple-topology.conf"), this.getClass.getSimpleName)
```

Comment:
We have `simpleTopology4Svs` that should take care of most things, you probably want to keep your custom transform and the manualStart one